CHANGELOG.md

```diff
@@ -60,7 +60,9 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - Added `replacingOccurrences(ofPattern:withTemplate:options:searchRange:)` as a more convenient way to replace patterns. [#901](https://github.com/SwifterSwift/SwifterSwift/pull/901) by [gurgeous](https://github.com/gurgeous)
 - **Measurement**
   - Added `.degrees(_:)`, `arcMinutes(_:)`, `arcSeconds(_:)`, `radians(_:)`, `gradians(_:)` and `revolutions(_:)`  to conveniently initialize measurement with corresponding unit. [#936](https://github.com/SwifterSwift/SwifterSwift/pull/936) by [Shiva Huang](https://github.com/ShivaHuang)
-
+- **Array**
+  - Added `append()` and `append(contentsOf` method overloads that can append optional element to `Array`. By [jevonmao](https://github.com/jevonmao)
```

Comment:
One last thing can you update the new names and the fact that they are not overloads anymore and also not on `Array`

Reply:
Done.